documentation/modules/exploit/linux/http/apache_hugegraph_gremlin_rce.md

```diff
@@ -0,0 +1,46 @@
+## Vulnerable Application
+This module exploits CVE-2024-27348 which is a Remote Code Execution (RCE) vulnerability that exists in
+Apache HugeGraph Server in versions before 1.3.0. An attacker can bypass the sandbox restrictions and achieve
+RCE through Gremlin, resulting in complete control over the server
+
+### Setup
+To install a vulnerable instance via docker run the following command:
+```
+docker run -itd --name=graph -p 8080:8080 hugegraph/hugegraph:1.0.0
+```
+
+## Verification Steps
+
+1. Start msfconsole
+1. Do: `use exploit/multi/http/apache_hugegraph_gremlin_rce`
```

Comment:
I'm assuming the module path was updated and some poin, as this seems to be outdated now:
```suggestion
1. Do: `use exploit/linux/http/apache_hugegraph_gremlin_rce`
```
![image](https://github.com/user-attachments/assets/7e1d85ed-b584-43ec-b312-62a2e8ac97cf)